quickref/qsharp-quick-reference.tex

```diff
@@ -165,22 +175,25 @@
     Call adjoint \newline operation & \texttt{Adjoint \emph{Name}(\emph{parameters})} \\
     Call controlled \newline operation & \texttt{Controlled \emph{Name}(\emph{controlQubits}, \emph{parameters})} \\
   \end{keysref}
-
   \begin{keysref}{Symbols and Variables}
     Declare immutable \newline symbol & \texttt{let \emph{name} = \emph{value}} \\
     Declare mutable \newline symbol (variable)  & \texttt{mutable \emph{name} = \emph{initialValue}} \\
     Update mutable \newline symbol (variable)   & \texttt{set \emph{name} = \emph{newValue}} \\
+    Apply-and-Reassign & \texttt{mutable \emph{name} = \emph{initialValue}} \newline
+    \texttt{for (\emph{index} in \emph{range}) \{ } \newline
+    \texttt{ set counter += \emph{index};} \texttt{\}} \\ 
   \end{keysref}
 
-  \columnbreak%\
-
   \begin{keysref}{Arrays}
     Allocation           & \texttt{mutable \emph{name} = new \emph{Type}[\emph{length}]} \\
     Length               & \texttt{Length(\emph{name})} \\
     k-th element         & \texttt{\emph{name}[k]} \newline NB: indices are 0-based  \\
     Array literal        & \texttt{[\emph{value0}, \emph{value1}, ...]} \newline
                            e.g.: \texttt{[true, false, true]} \\
-    Slicing (subarray)   & \texttt{\emph{name}[\emph{start}..\emph{end}]} \\
+    Slicing (subarray)   & \texttt{\emph{name}[\emph{start}...\emph{end}]} \newline
```

Comment:
This section gives the syntax but doesn't clarify what different types of syntax do - I think it would be more helpful if the left column had an example of an array we're slicing, and the right column listed both syntax used and the result you'll get. You can also highlight slicing with steps other than 1 this way very nicely.